tomviz/Pipeline.h

```diff
@@ -59,6 +59,14 @@ class Pipeline : public QObject
   /// Return true if the pipeline is currently being executed.
   bool isRunning();
 
+  /// Return true is the pipeline is currently paused.
+  bool isPaused() const;
+
+  /// The user operator has started/finished editing an operator
+  void startedEditingOp() { ++m_editingOperators; }
+  void finishedEditingOp() { --m_editingOperators; }
+  int editingOperators() { return m_editingOperators; }
```

Comment:
I think start and finished should be done through signals and then have a single public method ```isDraft``` or ```isBeingEdited```